types/src/connectors/google_drive.ts

```diff
@@ -0,0 +1,51 @@
+// Get the Table ID for a sheet within a Google Spreadsheet from the
+// Google-provided file ID and the ID of the sheet within the spreadsheet.
+export function getGoogleSheetTableId(
+  googleFileId: string,
+  googleSheetId: number
+): string {
+  return `google-spreadsheet-${googleFileId}-sheet-${googleSheetId}`;
+}
+
+// Get the Content Node ID for a sheet within a Google Spreadsheet from the
+// Google-provided file ID and the ID of the sheet within the spreadsheet.
+export function getGoogleSheetContentNodeInternalId(
```

Comment:
Why do we need this method given that it does exactly like `getGoogleSheetTableId`?

Reply:
I agree that naming here helps a lot, I'm find to keep it around.